src/redux/actions/authActions.js

```diff
@@ -35,6 +36,17 @@ export const toggleLogin = (payload) => {
   return { type: LOGIN, payload };
 };
 
+export const verifyUser = () => {
+    return (dispatch) => {
+     firebase.auth().onAuthStateChanged(user => {
+         if(user){
+           dispatch({type: LOGIN_SUCCESS});
+         } else {
+             dispatch({type: REGISTER});
```

Comment:
We forgot to handle else. We should add dispatch({type: VERIFY_FAILS}) or something similar and set it loggedIn to false. That way if the user disappears from the local session, he will be logged out immediately.